utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -1196,14 +1266,6 @@ int main(int argc, char **argv)
 
             if (reductionTypeCase)
             {
-                if(srcDescPtr->c == 3)
```

Comment:
why this is deleted?

Reply:
This was a mistake during merging with latest branch, reverted this.